code-snippets/monitoring.md

```diff
@@ -0,0 +1,31 @@
+# Liveness & Readiness probe
+
+Application in OpenShift can monitor itself using liveness probe. When the liveness probe detects an application is in broken state OpenShift automatically restart a pod with an application.
+
+The readiness probe is usually used when application starts and it is not ready to get requests yet. The readiness probe blocks incoming traffic of requests until application is fully ready.
```

Comment:
Readiness probe is usually used when an application... not ready to receive requests yet

The readiness probe delays incoming traffic until application is ready to process them